triton/tut/gpu.rst

```diff
@@ -312,19 +312,231 @@ on efficient data loading.
+
+    Section: Launch Statistics
+    -------------------------------- --------------- ---------------
+    Metric Name                          Metric Unit    Metric Value
+    -------------------------------- --------------- ---------------
+    Block Size                                                   128
+    Function Cache Configuration                     CachePreferNone
+    Grid Size                                                    512
+    Registers Per Thread             register/thread              21
+    Shared Memory Configuration Size            byte               0
+    Driver Shared Memory Per Block        byte/block               0
+    Dynamic Shared Memory Per Block       byte/block               0
+    Static Shared Memory Per Block        byte/block               0
+    Threads                                   thread           65536
+    Waves Per SM                                                0.40
+    -------------------------------- --------------- ---------------
+
+    Section: Occupancy
+    ------------------------------- ----------- ------------
+    Metric Name                     Metric Unit Metric Value
+    ------------------------------- ----------- ------------
+    Block Limit SM                        block           32
+    Block Limit Registers                 block           21
+    Block Limit Shared Mem                block           32
+    Block Limit Warps                     block           16
+    Theoretical Active Warps per SM        warp           64
+    Theoretical Occupancy                     %          100
+    Achieved Occupancy                        %        39.97
+    Achieved Active Warps Per SM           warp        25.58
+    ------------------------------- ----------- ------------
+
+    OPT   Estimated Speedup: 60.03%
+          This kernel's theoretical occupancy is not impacted by any block limit. The difference between calculated
+          theoretical (100.0%) and measured achieved occupancy (40.0%) can be the result of warp scheduling overheads
+          or workload imbalances during the kernel execution. Load imbalances can occur between warps within a block
+          as well as across blocks of the same kernel. See the CUDA Best Practices Guide
+          (https://docs.nvidia.com/cuda/cuda-c-best-practices-guide/index.html#occupancy) for more details on
+          optimizing occupancy.
+The profiling result suggests that the kernel grid is too small, meaning the workload is not sufficient to fully utilize the GPU's resources. Only 0.4 full waves across all SMs were active, indicating suboptimal GPU utilization. To improve performance, one may need to increase the grid size to better match the GPU's capabilities.
+
+For detailed guides on how to use ``ncu``, refer to the documentation: `NVIDIA Nsight Compute suite <https://docs.nvidia.com/nsight-compute/index.html>`__.
+
+
+.. admonition:: To be Deprecated
```

Comment:
I would remove everything about nvprof. New documentation is better and new tools provide more information than nvprof provides.